eks/clusterautoscaler/clusterautoscaler.go

```diff
@@ -89,6 +91,12 @@ func (c *ClusterAutoscaler) buildCommand() string {
 	return string(json)
 }
 
+func (c *ClusterAutoscaler) buildResources() string {
+	json, _ := json.Marshal(c.Config.Spec.ClusterAutoscaler.Resources)
```

Comment:
These are actually already included in the gotemplate from c.Config.Spec.ClusterAutoscaler on line 36.

Reply:
Got it, changed.